Makefile

```diff
@@ -0,0 +1,34 @@
+IMAGE_NAME = methodology/docusaurus:latest
+CONTAINER_NAME = open-edu-hub-methodology-bash
+OUTPUT_DIR = $$PWD/.output/methodology/
+
+.PHONY: all buildimg build run clean cleanall
+
+all: build
+
+buildimg:
+	docker build -f ./Dockerfile --tag $(IMAGE_NAME) .
+
+build: buildimg
+	@echo "Building content. This will take a while (several minutes) ..."
+	@mkdir -p $(OUTPUT_DIR)
+	docker run --rm -v $$PWD/:/content -v $(OUTPUT_DIR):/output $(IMAGE_NAME)
+	@echo "Content built in $(OUTPUT_DIR)"
+	@echo "Run `make serve` to start content server."
```

Comment:
This line is not printed when running `make build`. No idea why. Also, `make serve` is not needed, because `docker run ...` above already handles that, so you might want to consider removing the `serve` rule, unless you use it for something else and I don't see it.